code/modules/projectiles/guns/ballistic.dm

```diff
@@ -449,6 +449,7 @@
 			flip_cooldown = (world.time + 30)
 			user.visible_message("<span class='notice'>[user] spins the [src] around their finger by the trigger. That’s pretty badass.</span>")
 			playsound(src, 'sound/items/handling/ammobox_pickup.ogg', 20, FALSE)
+			return
```

Comment:
```suggestion
if((!HAS_TRAIT(user, TRAIT_CLUMSY)) || prob(50))
			return
```
(:

Reply:
They already have a 40% chance to shoot them in the foot when spinning, I personally don't feel they should additionally get a 50% chance to drop all their bullets.

Reply:
oh ok